src/verilog.y

```diff
@@ -1082,6 +1082,7 @@ BISONPRE_VERSION(3.7,%define api.header.include {"V3ParseBison.h"})
 //  Blank lines for type insertion
 //  Blank lines for type insertion
 //  Blank lines for type insertion
+//  Blank lines for type insertion
```

Comment:
This is ok, but FYI usually I add a bunch at once just to avoid updating so often.